cloud/services/domains.go

```diff
@@ -295,8 +296,38 @@ func removeElement(stringList []string, elemToRemove string) []string {
 	return stringList
 }
 
+func processLinodeMachine(ctx context.Context, cscope *scope.ClusterScope, eachMachine v1alpha2.LinodeMachine, dnsTTLSec int, subdomain string) ([]DNSOptions, error) {
+	// Look up the corresponding CAPI machine, see if its deleted.
```

Comment:
just for clarity, this comment should say `see if it is marked for deletion` I think?